rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -1379,9 +1379,14 @@ def get_blocknumber_by_time(
             closest: Literal['before', 'after'] = 'before',
     ) -> int:
         """Searches for the blocknumber of a specific timestamp
-
+        - Performs the blockscout api call by default first
+        - If RemoteError gets raised it queries etherscan
         May raise RemoteError
         """
+        if self.blockscout:
```

Comment:
when can this if be off? Do we allow the user to choose if blockscout should be queried?

Reply:
this will be false only for scroll. No, blockscout can't be configured to be queried but this is just the get block by timestamp function. This call is one of the failing from your logs. the other place where blockscout is called is in the eth staking logic.